hack/build-cross.sh

```diff
@@ -4,43 +4,32 @@
 STARTTIME=$(date +%s)
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
+host_arch="$(os::build::host_arch)"
 host_platform="$(os::build::host_platform)"
 
 # Set build tags for these binaries
 readonly OS_GOFLAGS_TAGS="include_gcs include_oss containers_image_openpgp"
 readonly OS_GOFLAGS_TAGS_$(os::build::platform_arch)="gssapi"
 
 # by default, build for these platforms
-platforms=(
-  linux/amd64
+platforms=( )
+for arch in "${OS_BUILD_SUPPORTED_ARCHES[@]}"; do
```

Comment:
The should be server platforms and should not be linux specific.  We will eventually build on non-linux this way.